bindgen/ir/context.rs

```diff
@@ -804,7 +804,7 @@ If you encounter an error missing from this list, please file an issue or a PR!"
             self.current_module
         );
 
-        self.items[(self.current_module.0).0]
+        self.items[self.current_module.0 .0]
```

Comment:
This change looks drive-by but alright.

Reply:
sorry, i forgot to include it in the description - i actually fixed a number of these too as part of this PR.  Updated.